### DISCUSSION TITLE
Enable/disable linting?

### DISCUSSION THREAD

**Discussion by Max-crosskey** [codefori/vscode-ibmi#1861] [Q&A]
Hi!

Recently started using the linter and it's great when creating new sources but I can't seem to find a way to enable or disable the linter? It's always on which is quite annoying when you open up a very old source that doesn't follow the rules. Is there a setting I'm missing to only enable the linter when compiling/saving/editing/browsing?

**Comment by worksofliam** [comment]
If you have created a rpglint for a library, it will apply to all **free members in that library.

There is no way to turn it off for specific members.

**Comment by Gregory1701** [comment]
Hey @Max-crosskey I've been working with the RPG Lint and trying to get it 'the way we want' before sharing with my team. And to Liams' comment, I find it actually a 'saving grace' that it will only fire on **Free members. As our source library is probably upwards of 90+ % not **Free and then the Linter would blow up the source, making it a hindrance rather than a benefit.

But on occasion, I do run across a **Free member where it does blow up my source and want to turn it off. It's not a perfect solution, but I would offer to you my solution to your question. When I want to 'turn it off', I just rename the rpglint.json to rpglintx.json. Then remove the x when I want to turn it back on.

Hope this helps.

Happy Coding,
Gregory
